napari/_tests/test_viewer.py

```diff
@@ -168,6 +168,11 @@ def test_screenshot(make_napari_viewer):
     # test size argument (and ensure it coerces to int)
     screenshot = viewer.screenshot(canvas_only=True, size=(20, 20.0))
     assert screenshot.shape == (20, 20, 4)
+    qtbot.waitUntil(
```

Comment:
Add an implementation/explanatory comment to clarify why we need this, since it is not immediately obvious in this context. By following the code, I think it's a proxy for detecting that the animation has finished - right? And we can't wait for the `QAbstractAnimation::finished` signal since we don't know if the animation is still accessible, as it is removed?

Reply:
could you check the comment text?

Reply:
looks great - thanks!